### DISCUSSION TITLE
Updating PIN documentation

### DISCUSSION THREAD

**Discussion by mores** [Pi4J/pi4j#382] [General]
I would like to update some PIN documentation....
1. there are 2 pages that are almost the same - can we just have 1 and if so which one ?
https://www.pi4j.com/getting-started/understanding-the-pins/
https://www.pi4j.com/documentation/pin-numbering/

2. I would like to replace the image with a html table like: https://randomnerdtutorials.com/raspberry-pi-pinout-gpios/

3. Example output from raspi-gpio get
When I run it I get a total of 53 GPIOs - why is that and how does that relate to physical numbering

**Comment by FDelporte** [comment]
Thanks @mores for this idea. My thoughts

1/ the getting-started page is part of a series of pages to guide people into Pi4J, so it is required. Maybe we could remove the table here with the numbering and just give one Gpio pin example, and refer to the other page for more details?

2/ yes image can be replaced and have it only on pin-numbering page. This will probably be also better for SEO. Would be good on this page to refer to api.pi4j.com as more board info is there, and that's where the image comes from: https://api.pi4j.com/board-information/MODEL_4_B

3/ I guess the SOC has more pins available, but only a limited list is connected to the header. I'm wondering if the compute module exposes more of those pins...
